network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/news_you_can_use.html

```diff
@@ -30,9 +30,13 @@ <h2 class="h4-heading mb-4">{% trans "News you can use" %}</h2>
             {% image first.get_meta_image fill-1000x500 %}
           </div>
           <div class="feature-content align-self-center">
-            {% with category=first.category.first %}
-              <a class="h6-heading d-block mb-2" href="/blog/category/{{ category.slug }}">{{ category }}</a>
-            {% endwith %}
+            {% if item.blog.category.count %}
```

Comment:
oops, missed this, but percy caught it:
```suggestion
            {% if first.category.count %}
```